include/RAJA/policy/openmp/policy.hpp

```diff
@@ -22,14 +22,32 @@
 
 #include "RAJA/policy/PolicyBase.hpp"
 
+#include <omp.h>
+
 namespace RAJA
 {
 namespace policy
 {
-
 namespace omp
 {
 
+namespace internal
+{
+    template <omp_sched_t Sched, int Chunk>
+    struct Schedule {
+        constexpr static omp_sched_t schedule = Sched;
```

Comment:
`schedule` will be negative if `Auto`
`chunk_size` will be negative if "default"